dlt/normalize/normalize.py

```diff
@@ -341,7 +341,7 @@ def spool_files(
 
     def spool_schema_files(self, load_id: str, schema: Schema, files: Sequence[str]) -> str:
         # normalized files will go here before being atomically renamed
-        self.load_storage.new_packages.create_package(load_id)
+        self.load_storage.new_packages.create_package(load_id, True)
```

Comment:
if package exists you must delete the previous package, there's a partial content there possibly so can easily do the double load. btw. a good catch! try to improve test so we have a double load. not sure how to provoke a normalizer error. maybe with mocking?